examples/webgpu_postprocessing_3dlut.html

```diff
@@ -0,0 +1,195 @@
+			import { RGBELoader } from 'three/addons/loaders/RGBELoader.js';
+			import { LUTCubeLoader } from 'three/addons/loaders/LUTCubeLoader.js';
+			import { LUT3dlLoader } from 'three/addons/loaders/LUT3dlLoader.js';
+			import { LUTImageLoader } from 'three/addons/loaders/LUTImageLoader.js';
+			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+
+			const params = {
+				lut: 'Bourbon 64.CUBE',
+				intensity: 1
+			};
+
+			const lutMap = {
+				'Bourbon 64.CUBE': null,
+				'Chemical 168.CUBE': null,
+				'Clayton 33.CUBE': null,
+				'Cubicle 99.CUBE': null,
+				'Remy 24.CUBE': null,
+				'Presetpro-Cinematic.3dl': null,
+				'NeutralLUT': null,
+				'B&WLUT': null,
+				'NightLUT': null
+			};
+
+			let gui;
+			let camera, scene, renderer;
+			let postProcessing, lutPass;
+
+			init();
+
+			async function init() {
+
+				const container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.25, 20 );
+				camera.position.set( - 1.8, 0.6, 2.7 );
+
+				scene = new THREE.Scene();
+
+				new RGBELoader()
+					.setPath( 'textures/equirectangular/' )
+					.load( 'royal_esplanade_1k.hdr', function ( texture ) {
+
+						texture.mapping = THREE.EquirectangularReflectionMapping;
+
+						scene.background = texture;
+						scene.environment = texture;
+
+						// model
+
+						const loader = new GLTFLoader().setPath( 'models/gltf/DamagedHelmet/glTF/' );
+						loader.load( 'DamagedHelmet.gltf', function ( gltf ) {
+
+							scene.add( gltf.scene );
+
+						} );
+
+					} );
+
+				const lutCubeLoader = new LUTCubeLoader();
+				const lutImageLoader = new LUTImageLoader();
+				const lut3dlLoader = new LUT3dlLoader();
+
+				for ( const name in lutMap ) {
+
+					if ( /\.CUBE$/i.test( name ) ) {
+
+						lutMap[ name ] = lutCubeLoader.loadAsync( 'luts/' + name );
+
+					} else if ( /\LUT$/i.test( name ) ) {
+			
+						lutMap[ name ] = lutImageLoader.loadAsync( `luts/${name}.png` );
+			
+					} else {
+
+						lutMap[ name ] = lut3dlLoader.loadAsync( 'luts/' + name );
+
+					}
+
+				}
+
+				const pendings = Object.values( lutMap );
+				await Promise.all( pendings );
+
+				for ( const name in lutMap ) {
+
+					lutMap[ name ] = await lutMap[ name ];
+
+				}
+
+				renderer = new THREE.WebGPURenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.setAnimationLoop( animate );
+				renderer.toneMapping = THREE.NoToneMapping;
+				renderer.outputColorSpace = THREE.LinearSRGBColorSpace;
+				container.appendChild( renderer.domElement );
+
+				// postprocessing
+
+				postProcessing = new THREE.PostProcessing( renderer );
+
+				const scenePass = pass( scene, camera );
+				const scenePassColor = scenePass.getTextureNode();
+
+				const outputPass = scenePassColor.toneMapping( THREE.ACESFilmicToneMapping ).linearTosRGB();
```

Comment:
@sunag LUT is a pass that comes after tone mapping and sRGB color space conversion. Right now, I had to disable the settings on renderer level and apply them as TSL in this line. I have the feeling this is not ideal.

Other pass like FXAA also require sRGB input so we need the same logic there as well. Should we make a separate `OutputNode` which extracts the renderer settings like the previous `OutputPass`?

Reply:
It seems `Renderer` does currently always a separate pass to apply tone mapping and color space conversion.

https://github.com/mrdoob/three.js/blob/3e8a091b30baf2e4369145789db91127349f1ddb/src/renderers/common/Renderer.js#L684-L688

This is actually not ideal. When doing post processing tone mapping and color space can be combined in the pass chain.

Besides, it is in general not preferable for the performance to abandon inline tone mapping and color space conversion since we always end up with an additional pass.

Reply:
As a first step I propose `OutputNode` so the renderer settings are honored and the node itself can freely added to the pass chain.

The inline tone mapping and color space conversion is a different topic and can be discussed separately. Doing it with a separate pass fixes #23019 but performance should not be neglected.

Reply:
> Um, this is actually not ideal. When doing post processing, tone mapping and color space can (and should) be combined in the pass chain.

This benefits refraction when used by avoiding double rendering of opaque objects improving performance, and increases the possibilities of sharing the same shaders as it reduces the final shader code for each material, consequently saves gpu memory. The benefits are quite considerable compared to the previous approach. https://github.com/mrdoob/three.js/pull/27880

I think some input needs sRGB we can do this conversion at the Node input, this would be even more reliable since all RenderTargers use the same color space.

Reply:
> The inline tone mapping and color space conversion is a different topic and can be discussed separately. Doing it with a separate pass fixes https://github.com/mrdoob/three.js/issues/23019 but performance should not be neglected.

I believe that neglecting performance would be a point of view, for example, if we focus on performance with opaque objects running on the screen without post-processing, inline tonemapping would make sense (although I can't see how applying sRGB and tone-mapping separately could harm performance so much, especially in the context of webgpu where frame buffers are also textures and we don't use global states). On the other hand, if we focus on a scene where we will have PBR, transparent and opaque materials in the same scene, and with the possibility of effects accessing the already rendered buffer, maintaining performance even with a lot of objects in the same scene, and in an easy way as the examples of backdrop and `viewportTexture()` show us and with PostProcessing, this is the way. The difference in the end is that WebGPURenderer uses post-processing as default, nothing different from that. All these resources are aligned with this new architecture, and if we use PostProcessing with a custom tone-mapping this pass does not need to be necessary. The question I asked myself was what the renderer is prioritizing as performance.

Reply:
Thanks for sharing the reference to https://github.com/mrdoob/three.js/pull/27880. I recall now the motivation behind the new approach and indeed it makes sense. 

But the graphic from the post assume sRGB/tone mapping comes after post processing which is not always true. I think this bit needs to be revisited.

> I think some input needs sRGB we can do this conversion at the Node input, this would be even more reliable since all RenderTargers use the same color space.

Conversion between color spaces is a back-and-forth operation. But how would you handle the tone mapping case? If a user sets up the 3D LUT effect, the renderer should apply tone mapping _before_ the updating colors with the LUT.

Reply:
> The difference in the end is that WebGPURenderer uses post-processing as default, nothing different from that. All these resources are aligned with this new architecture, and if we use PostProcessing with a custom tone-mapping this pass does not need to be necessary.

My point is if the app uses `PostProcessing` a separate output pass by the renderer is not really necessary. It's like always having `OutputPass` in the pass chain with the pervious composer. My hope was that we can get rid of this step with the new system, tbh.

Reply:
> My point is if the app uses PostProcessing a separate output pass by the renderer is not really necessary. It's like always having a separately executed OutputPass in the pass chain with the pervious composer. My hope was that we can get rid of this extra render pass with the new system, tbh.

Make sense, that's a good idea, I think we can do that in the PostProcessing class.